xp/mssql/Dockerfile

```diff
@@ -1,63 +1,65 @@
 # escape=`
+FROM mcr.microsoft.com/windows/servercore:ltsc2019
```

Comment:
Is er geen nativate mssql op server core?

Reply:
Hmm ja had misschien ook gekund, heb me hier laten leiden door de sitecoreops implementatie van Per Bering (zie wel dat ik hier die ben vergeten te mention-en)